### DISCUSSION TITLE
Assets of v2.0.6

### DISCUSSION THREAD

**Discussion by pzhlkj6612** [miurahr/aqtinstall#487] [Q&A]
-  [aqt.exe](https://github.com/miurahr/aqtinstall/releases/download/v2.0.6/aqt.exe)  10.8 MB  MD5: B98CF6E5BA1E97C8697B987ED44D3FF3
-  [aqt_x64.exe](https://github.com/miurahr/aqtinstall/releases/download/v2.0.6/aqt_x64.exe)  10.8 MB  MD5: B98CF6E5BA1E97C8697B987ED44D3FF3
-  aqt_x86.exe  9.36 MB

Duplicates?

**Comment by miurahr** [comment]
https://github.com/miurahr/aqtinstall/pull/471 make 32bit release. Did It change?
